.eslintignore

```diff
@@ -1,2 +1,3 @@
 !.storybook
 build
+storybook-static
```

Comment:
i keep running into this when i infrequently run storybook and don't clean it up..